docs/docs/tutorials/get-started/enable-passcode-sign-in.mdx

```diff
@@ -21,40 +19,9 @@ Depending on what kind of connector you want to enable, click the "Set Up" butto
 
 Logto has some built-in connectors which allows out-of-box usage:
 
-<Columns
-  columns={[
-    {
-      title: 'SMS connectors',
-      items: [
-        {
-          key: 'Twilio SMS',
-          link: 'https://github.com/logto-io/connectors/tree/master/packages/connector-twilio-sms',
-        },
-        {
-          key: 'Aliyun SMS',
-          link: 'https://github.com/logto-io/connectors/tree/master/packages/connector-aliyun-sms',
-        },
-      ],
-    },
-    {
-      title: 'Email connectors',
-      items: [
-        {
-          key: 'SendGrid Mail',
-          link: 'https://github.com/logto-io/connectors/tree/master/packages/connector-sendgrid-email',
-        },
-        {
-          key: 'Aliyun DM',
-          link: 'https://github.com/logto-io/connectors/tree/master/packages/connector-aliyun-dm',
-        },
-        {
-          key: 'SMTP',
-          link: 'https://github.com/logto-io/connectors/tree/master/packages/connector-smtp',
-        },
-      ],
-    },
-  ]}
-/>
+> Twilio SMS, Aliyun SMS, AWS SES, Tecent SMS, SendGrid Mail, Aliyun DM, SMTP, ...
```

Comment:
nit
```suggestion
> Twilio SMS, Aliyun SMS, AWS SES, Tecent SMS, SendGrid Mail, Aliyun DM, SMTP, any many more...
```

Reply:
idkw, "any(d) many more" looks not decent lol

Reply:
i prefer to keep it